src/base/error_code.h

```diff
@@ -1728,7 +1728,9 @@
 
 #define ER_CGW_INVALID_COLUMNS_SPECIFIED         -1352
 
-#define ER_LAST_ERROR                               -1353
+#define ER_UHOST_IP_ADDR_TOO_LONG         -1353
```

Comment:
1. Error message like` msg/en_US ` will be even added in other languages.
2. @kisoo-han @airnet73. What about **ER_UHOST_IP_ADDR_TOO_LONG** ?
Shall we check the name of macro?

Reply:
It looks good, I propose another **ER_UHOST_INVALID_IP_ADDR**.